### PR TITLE
CRDCDH-1253 Remove Data Submission Upload Types

### PR DESCRIPTION
**Pull request by Alejandro-Vega**
### Overview

Removed "Upload Types" within the Data Submission Dashboard. 

### Change Details (Specifics)

- Removed the Radio button options from Step 1 "Upload Metadata".
- Adjusted positioning and spacing for Choose Files button and label
- Added tooltip to "Upload Metadata" step title
- Removed any mention of `metadataIntention`
- Removed "Upload Type" column from the Data Activity table

### Related Ticket(s)

[CRDCDH-1253](https://tracker.nci.nih.gov/browse/CRDCDH-1253)

### CODE REVIEW

File: src/components/DataSubmissions/MetadataUpload.tsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { LoadingButton } from "@mui/lab";
 import { isEqual } from "lodash";
 import { VariantType } from "notistack";
 import { Button, Stack, Typography, styled } from "@mui/material";
-import RadioInput from "./RadioInput";
+import Tooltip from "../Tooltip";
 import { CREATE_BATCH, CreateBatchResp, UPDATE_BATCH, UpdateBatchResp } from "../../graphql";
 import { useAuthContext } from "../Contexts/AuthContext";
 import DeleteDialog from "../../content/dataSubmissions/DeleteDialog";
```
Comment (amattu2) [inline src/components/DataSubmissions/MetadataUpload.tsx:11]: @Alejandro-Vega Can you just confirm that the DeleteDialog is still used for something in MetadataUpload? It seems like it isn't used anymore.

**Review by amattu2 — CHANGES_REQUESTED** [review]
One potential unused import noted. Everything else LGTM.

File: src/components/DataSubmissions/MetadataUpload.tsx (hunk 1 above)
Comment (Alejandro-Vega) [reply src/components/DataSubmissions/MetadataUpload.tsx]: Good catch, removed the dialog.

**Review by amattu2 — APPROVED** [review]
LGTM. Didn't retest.

**Pull request merged**